File: CSharp/HtmlTest/Base/FormattedCloneTestWalker.cs
using HtmlWalker;
using HtmlWalker.Platform;
using System;

namespace HtmlTest.Base
{
    public class FormattedCloneTestWalker : TestWalker
    {
        public FormattedCloneTestWalker(WalkerPlatform platform) : base(platform)
            => Walkers.Add(new FormattedCloneWalker(Output));
    }
}

File: Java/htmlwalker/src/test/java/htmltest/base/FormattedCloneTestWalker.java
package htmltest.base;

import htmlwalker.FormattedCloneWalker;
import htmlwalker.platform.WalkerPlatform;

public class FormattedCloneTestWalker extends TestWalker
{
    public FormattedCloneTestWalker(WalkerPlatform platform, WalkerPlatform.IOptions options)
    {
        super(platform, options);
        walkers.add(new FormattedCloneWalker(output()));
    }
}
